src/Core/src/Handlers/Editor/EditorHandler.Android.cs

```diff
@@ -7,14 +7,12 @@
 
 namespace Microsoft.Maui.Handlers
 {
-	// TODO: NET8 issoto - Change the TPlatformView generic type to MauiAppCompatEditText
 	// This type adds support to the SelectionChanged event
-	public partial class EditorHandler : ViewHandler<IEditor, AppCompatEditText>
+	public partial class EditorHandler : ViewHandler<IEditor, MauiAppCompatEditText>
```

Comment:
Do we need to do this? Changing handler base types are very breaking. We can make the edit test public, but maybe this needs a bit of discussion just to make sure.